app/templates/_webpack.config.js

```diff
@@ -23,16 +24,16 @@ module.exports = {
         ]
     },
     plugins: [
+        new webpack.DefinePlugin({
+            'process.env.NODE_ENV': production ? '"production"' : '"development"'
```

Comment:
Some libraries (Vue.js, React) do use `process.env.NODE_ENV` property in their code. We need to set it to `"production"` in order to make sure some redundant debug code is removed from our bundles.

Reply:
True, I noticed that for Vue.js but can't we just use `new webpack.EnvironmentPlugin(['NODE_ENV'])` in pulgins?